tutorial/chapter_2_9.md

```diff
@@ -8,6 +8,13 @@ with just one keystroke?
 
 `F5` to the rescue.
 
+Note: in some setups the shortcut may be mapped to `Ctrl + F5` without the Fn
```

Comment:
Hey @jaipandya sorry for the delay. My apologies, it would appear that another plugin I have installed (PHP Companion) has hijacked the "F5" key binding and I completely forgot about it; I have reassigned the "F5" binding for something I must have used a lot in that plugin and have since forgotten about it... So in this case, the original text is accurate.